tkg/managementcomponents/types.go

```diff
@@ -144,4 +144,5 @@ type AkoOperatorConfig struct {
 	AviManagementClusterControlPlaneVipNetworkName string `yaml:"avi_management_cluster_control_plane_vip_network_name,omitempty"`
 	AviManagementClusterControlPlaneVipNetworkCIDR string `yaml:"avi_management_cluster_control_plane_vip_network_cidr,omitempty"`
 	AviControlPlaneHaProvider                      bool   `yaml:"avi_control_plane_ha_provider,omitempty"`
+	AviIngressNodeNetworkList                      string `yaml:"avi_ingress_node_network_list,omitempty"`
```

Comment:
This is not a string actually, you can refer this:
https://github.com/vmware-tanzu/load-balancer-operator-for-kubernetes/blob/main/api/v1alpha1/akodeploymentconfig_types.go#L265